src/components/VmDetails/cards/NicsCard/NicListItem.js

```diff
@@ -40,19 +40,31 @@ const NicListItem = ({ idPrefix, nic, vmStatus, vnicProfileList, isEditing, onEd
         </span>
       </div>
       <Grid>
-        <Row>
-          <Col cols={4} className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
+        <Row className={style['nics-row']}>
+          <Col className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
```

Comment:
The reason is that the IPv4's max-width will always be 33.333%, and IPv6, respectively, 66.666%. This does not work when the screen becomes too narrow (max-width should be 100%), but that will not happen because flex-grow is 0. 
Below is the change with changed flex-grow.
Before: 
![screenshot-localhost-3000-2019 06 26-16-08-21](https://user-images.githubusercontent.com/25762021/60211583-5182fd00-982d-11e9-82d2-c05c8654b6fc.png)

After:
![screenshot-localhost-3000-2019 06 26-16-03-24](https://user-images.githubusercontent.com/25762021/60211596-58117480-982d-11e9-9214-c434367f5e05.png)

Reply:
I see what you're saying.  I think there is a better way to tweak the `Col` behavior while retaining the `cols={X}` attribute.  I'll send you a patch on chat to look at.

Reply:
Cool.  That patch looks good.